app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
https://github.com/slim-template/html2slim

not perfect, but good approximation